docs/base-account/reference/core/provider-rpc-methods/wallet_connect.mdx

```diff
@@ -66,16 +112,55 @@ Whether the wallet is connected.
   }]
 }
 ```
+
+```json With signInWithEthereum Capability
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "method": "wallet_connect",
+  "params": [{
+    "version": "1",
+    "capabilities": {
+      "signInWithEthereum": {
+        "nonce": "abc123def456",
+        "chainId": "0x2105"
+      }
+    }
+  }]
+}
+```
 </RequestExample>
 
 <ResponseExample>
-```json Success Response
+```json Basic Connection Response
 {
   "id": 1,
   "jsonrpc": "2.0",
   "result": {
-    "accounts": ["0x407d73d8a49eeb85d32cf465507dd71d507100c1"],
-    "chainId": "0x1",
+    "accounts": [{
+      "address": "0x407d73d8a49eeb85d32cf465507dd71d507100c1"
+    }],
+    "chainId": "0x2105",
+    "isConnected": true
+  }
+}
+```
+
+```json signInWithEthereum Response
```

Comment:
should consider just linking out to the SIWE capability page so that we don't have to maintain this in two places